Dockerfile

```diff
@@ -1,21 +1,50 @@
 FROM r-base
 
 RUN apt-get update \
-    && apt-get install -y wget unzip tar python python3-pip python2 sed bedtools libcurl4-gnutls-dev libxml2-dev libssl-dev gawk \
+    && apt-get install -y wget unzip tar python2 python3 python3-pip  sed bedtools libcurl4-gnutls-dev libxml2-dev libssl-dev gawk \
     && rm -rf /var/lib/apt/lists/*
+RUN wget https://bootstrap.pypa.io/pip/2.7/get-pip.py \
+    && python2 get-pip.py \
+    && rm get-pip.py
 RUN mkdir /gwas-tools
 ENV PATH="/gwas-tools:${PATH}"
 WORKDIR /gwas-tools
-RUN R -e "install.packages(c('mvtnorm', 'corpcor', 'tidyverse', 'magrittr', 'BiocManager'), repos = 'http://cran.us.r-project.org')"
-RUN R -e "BiocManager::install(c('martini','BioNet'))"
+RUN R -e "install.packages(c('mvtnorm', 'corpcor', 'tidyverse', 'magrittr', 'BiocManager', 'cowplot'), repos = 'http://cran.us.r-project.org')"
+RUN R -e "BiocManager::install(c('martini','BioNet','twilight'))"
+#RUN wget http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool_v0.7.5_x86_64.tgz \
+#    && tar -zxvf gtool_v0.7.5_x86_64.tgz
+#RUN wget https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_x86_64_static.tgz \
+#    && tar -zxvf impute_v2.3.2_x86_64_static.tgz && mv impute_v2.3.2_x86_64_static/impute2 .
+#RUN wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/liftOver && chmod a+x liftOver
 RUN wget https://github.com/bedops/bedops/releases/download/v2.4.35/bedops_linux_x86_64-v2.4.35.tar.bz2 \
     && tar jxvf bedops_linux_x86_64-v2.4.35.tar.bz2 \
     && cp bin/* .
+RUN wget https://github.com/YuanlongLiu/SigMod/raw/master/SigMod_v2.zip \
+    && unzip SigMod_v2.zip
+RUN wget https://github.com/raphael-group/hotnet2/archive/refs/tags/v1.2.1.tar.gz \
```

Comment:
I don't think we can include HotNet2. This should be commented out.

Reply:
This is no distribution of HotNet2 itself, just a link to were to download it. If the authors stop distributing it, this line won't work anymore for example.
Also, the [copyright licence](https://github.com/raphael-group/hotnet2/blob/master/LICENSE) in their repo specify "Permission to use this software, and any documentation, for non-commercial academic research purposes" as long as the specified terms and conditions are respected which I think we do

Reply:
The Dockerfile is automatically built on DockerHub, and downloadable by everyone. The end user doesn't need to build it themselves. I am not a copyright expert, but I am not sure we should expect the end user to stick to a license they haven't read and agreed to...

Reply:
The automatic built isn't an obligation, a Docker Pro, Team or Business subscription is in fact required to do so. 

Providing just the Dockerfile into gwas-tools repo for the end user to built it is not infringing any license. The end user is also responsible to check the licenses of the tools included as long as it is explicit in the repo, for example in the README